pkg/release-controller/release_info.go

```diff
@@ -43,12 +42,12 @@ func NewCachingReleaseInfo(info ReleaseInfo, size int64, architecture string) Re
 				var iArr []BugDetails
 				iArr, err = info.Bugs(parts[1], parts[2])
```

Comment:
`info.Bugs()` no longer returns an integer array, so having `iArr` and `iArrByte` is incredibly confusing!

Reply:
Changed to bugDetailsArr and bugDetailsByte.